src/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+  
+.stars__string {
+  height: 16px;
+  display: flex;
+}
+  
+.stars__star {
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("/images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+  
+.stars__star:last-child {
+  margin-right: 0;
+}
+  
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(1) {
+  background-image: url("/images/star-active.svg");
+}
+  
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(2) {
+  background-image: url("/images/star-active.svg");
+}
+  
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(3),
+.stars--5 .stars__star:nth-child(3) {
+  background-image: url("/images/star-active.svg");
+}
+  
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(4) {
+  background-image: url("/images/star-active.svg");
+}
+  
+.stars--5 .stars__star:nth-child(5) {
+  background-image: url("/images/star-active.svg");
+}

```

Comment:
![image](https://user-images.githubusercontent.com/38065883/120785399-c42aa600-c535-11eb-9314-bd68fe61652e.png)

Reply:
wow this is cool!)